packages/react-table/src/components/Table/EditableSelectInputCell.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import { Select, SelectOptionObject, SelectProps } from '@patternfly/react-core';
+import inlineStyles from '@patternfly/react-styles/css/components/InlineEdit/inline-edit';
+import formStyles from '@patternfly/react-styles/css/components/Form/form';
+
+// eslint-disable-next-line @typescript-eslint/interface-name-prefix
+export interface IEditableSelectInputCell extends Omit<React.HTMLProps<HTMLElement | HTMLDivElement>, 'onSelect'> {
+  /** Row index of this select input cell */
+  rowIndex: number;
+  /** Cell index of this select input cell */
+  cellIndex: number;
+  /** Data structure containing:
```

Comment:
@redallen is there a way to format these so it displays better in the docs?  It is just one line right now, not very readable.

Reply:
Unfortunately not -- but we can look into translating "\n" into `<br />` or something like that.